### PR TITLE
Image thumbnail support

### PR DESCRIPTION
**Pull request by shefalijoshi**
Resolves VIPEROMCT-350

Add thumbnail metadata for image type objects in the object provider
Add thumbnail format that changes the full size image url into the thumbnail URL.

*KNOWN ISSUE*: VIPERGDS-1585

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?
* [x] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Comment by unlikelyzero** [comment]
To test
- [ ] image thumbnails. Regression test and verify that cannot draw on thumbnail.
- [ ] all cams 16x display layout. Size and resize to increase to trigger the thumbnail view inside of a display layout
- [ ] 2 hours of images

**Review by unlikelyzero — CHANGES_REQUESTED** [review]
Let's find the list of things that we talked about in planning

**Review by ozyx — APPROVED** [review]
LGTM

File: src/providers/object-provider.js [hunk 1]
```diff
@@ -346,6 +346,19 @@ export default class YamcsObjectProvider {
                 }]
             }
         };
+
+        if (this.#isImage(obj)) {
+            obj.telemetry.values.push({
+                name: 'VIPER Image Thumbnail',
```
Comment (akhenry) [inline src/providers/object-provider.js:352]: Let's remove the reference to VIPER here.

**Review by akhenry — CHANGES_REQUESTED** [review]

File: src/providers/object-provider.js (hunk 1 above)
Comment (shefalijoshi) [reply src/providers/object-provider.js:352]: OK What do you want to call it? Just Image Thumbnail?

**Review by akhenry — APPROVED** [review]

**Review by unlikelyzero — APPROVED** [review]

**Pull request merged**